web/src/components/feature/Input.tsx

```diff
@@ -33,7 +34,7 @@ const FluidPackage = styled(FlexContainer)`
   }
 `
 
-export const CalculateFluid = ({
+export const Input = ({
```

Comment:
I'm not a fan of naming things input, because it's to generic. By looking at only the name, I would not be able to guess if this was a dropdown, a textfield, our package input dialog or our main input area. And text searches would get a number of falls positives. Could we come up with something more specific? Like CalculationInput perhaps? InputCard, maybe, though I'm not sure the name should refer to a specific component we've used. CalculationConfigs? InputAndRun?

Reply:
CalculationInput is good :slightly_smiling_face: